src/main/java/me/zeroeightsix/kami/gui/rgui/component/SettingSlider.kt

```diff
@@ -65,10 +65,10 @@ class SettingSlider(val setting: NumberSetting<*>) : Slider(setting.name, 0.0, s
             if (!listening) {
                 listening = true
                 value = 0.0
-                name = "0"
+                name.setValue("0")
             } else {
                 listening = false
-                name = originalName
+                name.setValue(originalName.value)
```

Comment:
Using getters and setters is really ehhh

Reply:
This stems from your previous comment, this was me blindly fixing errors without looking to where they actually came from.

Reply:
What is this actually for, why are the setting names being changed in this way?

Reply:
It's much easier?